<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWGenWData</class>
 <widget class="QMainWindow" name="MWGenWData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MWGenWData</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbWave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>波形选择</string>
    </property>
    <widget class="QRadioButton" name="rbSin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>正弦波</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbSquare">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>方波</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbSaw">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>锯齿波</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbTri">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>三角波</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbCus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义</string>
     </property>
    </widget>
   </widget>
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>571</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbPara">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>参数设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenWData">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存数据文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>94</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
